### DISCUSSION TITLE
About the flang compiler in LLVM 11

### DISCUSSION THREAD

**Discussion by zhiyuanzhai** [Homebrew/discussions#143] [Everyday usage] [answered]
Is there any plan on adding flang, the new fortran compiler included in LLVM 11, to llvm package as a build flag, or as an individual package?

**Comment by SMillerDev** [comment]
You seem to be planning it, here's some documentation to help you get started: https://docs.brew.sh/Updating-Software-in-Homebrew

**Comment by fxcoudert** [comment, marked as answer]
We decided against it for now: https://github.com/Homebrew/homebrew-core/pull/63081
You can read the rationale for that choice in the linked PR

**Reply by zhiyuanzhai** [reply]
Ok, thanks for your information.
Maybe it's better to wait until flang compiler is finally finished.

**Comment by carlocab** [comment]
flang is currently not suitable for homebrew/core, but it now builds against Homebrew llvm using [the formula](https://github.com/carlocab/homebrew-personal/blob/main/Formula/flang.rb) in my tap: 
```
brew install carlocab/personal/flang
```
It takes a horrendously long time, so I suggest passing the `-v` flag. 

It won't make it go any faster, but it made waiting for it more bearable for me for some reason. Ninja also seemed faster than make, but I don't mind changing it back to make if you have a strong opinion about it.

**Reply by jonchang** [reply]
Have you seen this blog post about bottling with github releases? https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/

**Reply by carlocab** [reply]
I have. I'd even read your earlier blog post about publishing bottles on bintray. Thanks for the reminder. I'll find time to do this over the next few days.
